e2e/nomad09upgrade/docker.nomad

```diff
@@ -6,8 +6,7 @@ job "sleep" {
       driver = "docker"
 
       config {
-        image = "busybox"
-        args = ["sleep", "10000"]
+        image = "redis"
```

Comment:
Might want to add a tag so there's a chance of reusing an already downloaded image (unless you want to force downloads).